xarray/tests/test_tutorial.py

```diff
@@ -2,29 +2,51 @@
 
 import pytest
 
-from xarray import DataArray, tutorial
-from xarray.tests import assert_identical, network
+from xarray import DataArray, DataTree, tutorial
+from xarray.testing import assert_identical
+from xarray.tests import network
+
+
+@pytest.fixture(autouse=True, name="testfile")
+def setUp():
+    yield "tiny"
 
 
 @network
 class TestLoadDataset:
-    @pytest.fixture(autouse=True)
-    def setUp(self):
-        self.testfile = "tiny"
-
-    def test_download_from_github(self, tmp_path) -> None:
+    def test_download_from_github(self, testfile, tmp_path) -> None:
         cache_dir = tmp_path / tutorial._default_cache_dir_name
-        ds = tutorial.open_dataset(self.testfile, cache_dir=cache_dir).load()
+        ds = tutorial.open_dataset(testfile, cache_dir=cache_dir).load()
```

Comment:
it's probably better to just hard-code the dataset name into the test, there's no point in parametrizing this (to be clear, this part of the test suite is pretty old):
```suggestion
        ds = tutorial.open_dataset("tiny", cache_dir=cache_dir).load()
```